modules/edu.gemini.seqexec.model/src/test/scala/edu/gemini/seqexec/model/BoopicklingSpec.scala

```diff
@@ -17,7 +17,8 @@ object SharedModelArbitraries {
   def sequencesQueueArb[A](implicit arb: Arbitrary[A]): Arbitrary[SequencesQueue[A]] = Arbitrary {
     for {
       b <- Gen.listOfN[A](maxListSize, arb.arbitrary)
-    } yield SequencesQueue(Conditions.default, b)
+      o <- Gen.option(Gen.alphaStr)
+    } yield SequencesQueue(Conditions.default, o, b)
```

Comment:
This is going to increase the number of generated tests by a lot. We'll have to control the number of tests in another PR, but until then I don't think there will be a quick fix to make the tests on Travis pass.